starter-code/src/merge-sort.js

```diff
@@ -1,15 +1,15 @@
 function mergeSort(array) {
   if(array.length < 2) { return array }
 
-  var middle  = Math.floor(array.length / 2);
-  var left    = array.slice(0, middle);
-  var right   = array.slice(middle);
+  const middle  = Math.floor(array.length / 2);
+  const left    = array.slice(0, middle);
+  const right   = array.slice(middle);
 
   return sortHalves(mergeSort(left), mergeSort(right));
 }
 
 function sortHalves(left, right) {
-  var array = [];
+  let array = [];
```

Comment:
You can use `const` because, even though we change the content of the array, the array itself stays the same. There are indeed no reassignment.